gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/java/io/gravitee/rest/api/portal/rest/resource/ApplicationAlertResource.java

```diff
@@ -83,14 +83,18 @@ public Response updateAlert(
         final UpdateAlertTriggerEntity updateAlertTriggerEntity = alertMapper.convertToUpdate(alertInput);
         updateAlertTriggerEntity.setId(alertId);
 
-        final AlertTriggerEntity updated = applicationAlertService.update(applicationId, updateAlertTriggerEntity);
+        final AlertTriggerEntity updated = applicationAlertService.update(
+            GraviteeContext.getExecutionContext(),
+            applicationId,
+            updateAlertTriggerEntity
+        );
 
         Alert alert = alertMapper.convert(updated);
         return Response.ok(alert).build();
     }
 
     private void checkPlugins() {
-        AlertStatusEntity alertStatus = applicationAlertService.getStatus();
+        AlertStatusEntity alertStatus = applicationAlertService.getStatus(GraviteeContext.getExecutionContext());
```

Comment:
Could you pass the execution context as a parameter to instantiate it only once ?

Reply:
:+1: